addons/mail/models/mail_channel.py

```diff
@@ -430,7 +431,7 @@ def _send_guidelines(self, partners):
         return True
 
     def _update_moderation_email(self, emails, status):
-        """ This method adds emails into either white or black of the channel list of emails 
+        """ This method adds emails into either white or black of the channel list of emails
```

Comment:
Please refrain from linting lines that your commit does not change, as it adds noise and messes up blame history ;-)